src/modules/docker_context.rs

```diff
@@ -77,3 +77,186 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         _ => None,
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::test::ModuleRenderer;
+    use ansi_term::Color;
+    use std::fs::File;
+    use std::io::{self, Write};
+
+    #[test]
+    #[ignore]
+    fn only_trigger_when_docker_config_exists() -> io::Result<()> {
```

Comment:
Is there any reason you can't un-ignore this test and do it in an empty temporary directory?

Reply:
The only reason is that I forgot when I was testing locally, have fixed that now.